Framework/API/test/FunctionFactoryTest.h

```diff
@@ -433,9 +433,9 @@ class FunctionFactoryTest : public CxxTest::TestSuite {
   void test_MultiDomainFunction_creation_moreComplex() {
     const std::string fnString =
         "composite=MultiDomainFunction,NumDeriv=true;(name=FunctionFactoryTest_"
-        "FunctA,a0=0,a1=0.5;name=FunctionFactoryTest_FunctB,b0=0.1,b1=0.2,ties="
+        "FunctA,a0=0,a1=0.5;name=FunctionFactoryTest_FunctB,b0=0.1,ties="
```

Comment:
It's not obvious to me why these values needed to change. Is something in the test dependent on them?